Algorithms/Graphs/Bellmanford.c

```diff
@@ -0,0 +1,134 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<string.h>
+#include<limits.h>/*for assigning maximum integer for all distances of vertices except for source vertex in function BellmanFord()*/
+//Name- Jyoti Mishra(@jstjyoti)
+//E mail-jyotimishra29091997@gmail.com
+/*Bellmanford's algorithm is a dynamic programming algorithm which can give all possible shortest part from a given source vertex to all the vertex in the graph containing negative or positive weighted edges and not having any negative weight cycle(a cycle of graph resulting negative weight as whole).It is a dynamic programming algorithm as need to use previous solutions of subproblems becuase we don't know future paths at the begining */

```

Comment:
Then what does the **Warshall's algorithm** do?

Reply:
Also, It is not clear how is it a Dynamic Programming algorithm?

Reply:
Okay I am Explaining in comment section.